doc/tutorial.md

```diff
@@ -0,0 +1,213 @@
+# Tutorial
+
+## Introduction
+This tutorial shows the steps for creating and instantiating a Dlite entity and populating it with data. For this, we will use an example `csv` file containing data corresponding to time measurements from a solar panel. The example file has the following format:
+
+
+| time                 | Impp [A] | Isc [A] | MPP [W] | Vmpp [V] | Voc [V] |
+|----------------------|----------|---------|---------|----------|---------|
+| 2022-08-01 00:41:00  |  1.188   |   1.25  |  38.182 |   32.14  |  37.33  |
+|  ...                 |   ...    |   ...   |   ...   |    ...   |   ...   |
+
+
+
+## Writing a DLite Entity
+Let us write an Entity representing our solar panel measurement. We start by creating a `json` file called `myEntity.json`.
+
+### **Step 1**: Giving it a unique identifier
+Firstly, we must provide a unique identifier for our Entity. There are several ways of doing this:
+
+#### **1.  Uniform Resource Identifier (URI)**
+We can create and assign a unique Uniform Resource Identifier (URI). An URI has the form **namespace/version/nameOfEntity**, and it can be either written directly in the json file:
+
+```json
+"uri": "http://www.ontotrans.eu/0.1/solarPanelMeasurement",
+```
+or one can give the **namespace**, **version**, and **name** of our Entity as separate fields:
+
+```json
+"name": "solarPanelMeasurement",
+"namespace": "http://www.ontotrans.eu/",
+"version": "0.1"
+```
+
+#### **2. Universally Unique Identifier (UUID)**
+Alternatively, we can generate and assign a universally unique identifier (UUID). In Python, this can be achieved by the use of the [UUID module](https://docs.python.org/3/library/uuid.html):
+
+```python
+import uuid
+
+# Generate a random UUID
+entity_uuid = uuid.uuid4()
+
+# Open the json file and write to it the uuid (note )
+with open("myEntity.json", "w") as entity_file:
+    entity_file.write('"uuid": {}'.format(entity_uuid))
+```
+Our json file now contains the following:
+
+```json
+"uuid": <uuid>
+```
+where `<uuid>` is a unique identifier for our Entity. 
+
+
+### **Step 2**: Providing a link to the metadata
+Let us assume that we have followed option 1 and provided a URI. The next step consists in adding a **meta** field that will link the Entity to its metadata. In DLite, all Entities are instances of the *EntitySchema*. The *EntitySchema* can be understood as the structure of the Entity. The **meta** field provides a URI for the EntitySchema. For a schematic overview of the DLite data structures and further detail, see [Concepts section of the DLite User Guide](https://sintef.github.io/dlite/user_guide/concepts.html). We add this information to our json file in the following manner:
```

Comment:
I think we should remove the sentence "The *EntitySchema* can be understood as the structure of the Entity". Or, alternatively, rewrite it as "The EntitySchema defines the structure of the Entity". 

My suggestion would be to write 

_In DLite, all Entities are instances of the *EntitySchema*. Therefore, in the **meta** field of our Entity, we provide the URI of the EntitySchema:

```json
"uri": "http://www.ontotrans.eu/0.1/solarPanelMeasurement",
"meta": "http://onto-ns.com/meta/0.3/EntitySchema",
```
For a schematic overview of the DLite data structures and further detail, see [Concepts section of the DLite User Guide](https://sintef.github.io/dlite/user_guide/concepts.html)._

And then proceed with Step 3. What do you think? My concern is that in trying to simplify it, we end up writing something that is a bit imprecise and can lead to confusion/misinterpretation.

Reply:
So, I specifically showed that sentence to Jesper and he said it was ok. But I like better your suggestion with "defines the structure". In fact, that's what that sentence was supposed to mean.

> *all Entities are instances of the EntitySchema. Therefore, in the meta field of our Entity, we provide the URI*
 > > it's not a 'therefore', is it?

Reply:
Ok, nice. I meant "therefore" as an explanation to why the "meta" field is set to the EntitySchema URI. But you're right, the "therefore" is not strictly necessary.